plugins/services/src/js/service-configuration/ServiceGeneralConfigSection.js

```diff
@@ -0,0 +1,274 @@
+import React from 'react';
```

Comment:
This files is `ServiceConfigGeneralSectionDisplay.js` and `GeneralServiceConfigSection.js` combined